packages/@okta/i18n/dist/json/country_nb.json

```diff
@@ -0,0 +1 @@
ne","FR":"Frankrike","GA":"Gabon","GB":"Storbritannia (UK)","GD":"Grenada","GE":"Georgia","GF":"Fransk Guiana","GG":"Guernsey","GH":"Ghana","GI":"Gibraltar","GL":"Grønland","GM":"Gambia","GN":"Guinea","GP":"Guadeloupe","GQ":"Ekvatorial-Guinea","GR":"Hellas","GS":"Sør-Georgia og Sør-Sandwich-øyene","GT":"Guatemala","GU":"Guam","GW":"Guinea-Bissau","GY":"Guyana","HK":"Hong Kong","HM":"Heard- og McDonald-øyene","HN":"Honduras","HR":"Kroatia","HT":"Haiti","HU":"Ungarn","ID":"Indonesia","IE":"Irland","IL":"Israel","IN":"India","IO":"Det britiske territoriet i Indiahavet","IQ":"Irak","IR":"Iran","IS":"Island","IT":"Italia","JE":"Jersey","JM":"Jamaica","JO":"Jordan","JP":"Japan","KE":"Kenya","KG":"Kirgisistan","KH":"Kambodsja","KI":"Kiribati","KM":"Komorene","KN":"Saint Kitts and Nevis","KP":"Nord-Korea","KR":"Sør-Korea","KW":"Kuwait","KY":"Cayman-øyene","KZ":"Kasakhstan","LA":"Laos","LB":"Libanon","LC":"Saint Lucia","LI":"Liechtenstein","LK":"Sri Lanka","LR":"Liberia","LS":"Lesotho","LT":"Litauen","LU":"Luxembourg","LV":"Latvia","LY":"Libya","MA":"Marokko","MC":"Monaco","MD":"Moldova","ME":"Montenegro","MG":"Madagaskar","MH":"Marshall-øyene","MK":"Makedonia","ML":"Mali","MM":"Myanmar","MN":"Mongolia","MO":"Macao","MP":"Nord-Marianene","MQ":"Martinique","MR":"Mauretania","MS":"Montserrat","MT":"Malta","MU":"Mauritius","MV":"Maldivene","MW":"Malawi","MX":"Mexico","MY":"Malaysia","MZ":"Mosambik","NA":"Namibia","NC":"Ny-Caledonia","NE":"Niger","NF":"Norfolkøya","NG":"Nigeria","NI":"Nicaragua","NL":"Nederland","NO":"Norge","NP":"Nepal","NR":"Nauru","NU":"Niue","NZ":"New Zealand","OM":"Oman","PA":"Panama","PE":"Peru","PF":"Fransk Polynesia","PG":"Papua New Guinea","PH":"Filippinene","PK":"Pakistan","PL":"Polen","PM":"Saint Pierre og Miquelon","PN":"Pitcairn","PR":"Puerto Rico","PS":"Palestina","PT":"Portugal","PW":"Palau","PY":"Paraguay","QA":"Qatar","RE":"Réunion","RO":"Romania","RS":"Serbia","RU":"Russland","RW":"Rwanda","SA":"Saudi-Arabia","SB":"Solomon-øyene","SC":"Seychellene","SD":"Sudan","SE":"Sverige","SG":"Singapore","SH":"St. Helena","SI":"Slovenia","SJ":"Svalbard og Jan Mayen","SK":"Slovakia","SL":"Sierra Leone","SM":"San Marino","SN":"Senegal","SO":"Somalia","SR":"Surinam","SS":"Sør-Sudan","ST":"São Tomé og Príncipeafrica. kgm","SV":"El Salvador","SY":"Syria","SZ":"Swaziland","TC":"Turks og Caicos-øyene","TD":"Tchad","TF":"De franske sørterritorier","TG":"Togo","TH":"Thailand","TJ":"Tadsjikistan","TK":"Tokelau","TL":"Øst-Timor","TM":"Turkmenistan","TN":"Tunisia","TO":"Tonga","TR":"Tyrkia","TT":"Trinidad og Tobago","TV":"Tuvalu","TW":"Taiwan","TZ":"Tanzania","UA":"Ukraina","UG":"Uganda","UM":"USAs ytre småøyer","US":"USA","UY":"Uruguay","UZ":"Usbekistan","VA":"Vatikanstaten","VC":"Saint Vincent og Grenadinene","VE":"Republikken Venezuela","VG":"De britiske Jomfruøyer","VI":"De amerikanske Jomfruøyer","VN":"Vietnam","VU":"Vanuatu","WF":"Wallis- og Futunaøyene","WS":"Samoa","YE":"Yemen","YT":"Mayotte","ZA":"Sør-Afrika","ZM":"Zambia","ZW":"Zimbabwe"}
```

Comment:
Not a blocker for this one. But, should we modify our script to add a new line at the end of this JSON?

Reply:
we could... I created https://oktainc.atlassian.net/browse/OKTA-187554